### ISSUE TITLE
I want a rule that will check use of `argnames` and `argvalues` in `@pytest.mark.parametrize`

### ISSUE THREAD

**Issue by M1troll** [m-burst/flake8-pytest-style#228]
# Rule request

## Description

Hi! Thank you for your plugin!

In my opinion,  to plugin should be added the rule to check use of `argnames` and `argvalues` in `@pytest.mark.parametrize`.
Please see examples:

### Bad Code:
```python
@pytest.mark.parametrize(
    ["template", "subject",  "status"],
    [
        pytest.param(
            SmallTemplate,
            "subject1",
            "confirmed",
        ),
        pytest.param(
            BigTemplate,
            "subject2",
            "requested",
        ),
    ],
    indirect=["template"],
)
def test_template(...):
    ...
```
### Good code:
```python
@pytest.mark.parametrize(
    argnames=["template", "subject",  "status"],
    argvalues=[
        pytest.param(
            SmallTemplate,
            "subject1",
            "confirmed",
        ),
        pytest.param(
            BigTemplate,
            "subject2",
            "requested",
        ),
    ],
    indirect=["template"],
)
def test_template(...):
    ...
```

## Rationale

I think using `arnames` and `argvalues` makes the code more readable and understandable. Especially when we use an indirect argument in `@pytest.mark.parametrize`. Since recently I have been using them and I think that my code has become more beautiful. Maybe I will find like-minded people in this)

**Comment by m-burst** [comment]
Hi @M1troll,

Thanks for opening this.
I personally do not feel that enforcing named arguments in all `@pytest.mark.parametrize` decorators is the way to go, as there are simple one-line cases where named arguments would be extraneous.

I'm closing this for now, but I might think about it again if the issue gets traction.
